app/helpers/blacklight_helper.rb

```diff
@@ -19,18 +19,51 @@ def wildcard_char_strip(solr_parameters)
     solr_parameters[:q] = solr_parameters[:q].delete('?')
   end
 
+  def prepare_left_anchor_search(solr_parameters)
+    return unless left_anchor_search?(solr_parameters)
+    left_anchor_escape_whitespace(solr_parameters)
+    left_anchor_add_wildcard(solr_parameters)
+  end
+
+  def left_anchor_query(solr_parameters)
+    if Flipflop.json_query_dsl?
+      solr_parameters.dig('json', 'query', 'bool', 'must', 0, :edismax, :query)
```

Comment:
I'm wondering about the `0` here.  Will this only work for the first search term (i.e. if a user did a left-anchor search in the second input in the advanced search form, would it still work)?

In the `SearchBuilder`, I noticed we [do a map](https://github.com/pulibrary/orangelight/blob/4427e81247c62bf51a9db29eeedd958c44cec966/app/models/search_builder.rb#L24-L26) to catch everything in the `must` array.

Reply:
Ah, I suspect you're right, thank you!